tests/test_docker_latex.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shutil
+import unittest
+import logging
+
+import quizgen.latex
+import quizgen.pdf
+import quizgen.util.dirent
+import quizgen.util.json
+import tests.base
+
+class TestDockerPdfConversion(tests.base.BaseTest):
+    def test_quiz_to_pdf(self):
+        if not quizgen.latex.is_available(use_docker = True):
+            self.skipTest("Docker is not installed or not running")
+
+        source_quiz_path = tests.base.discover_good_quiz_files()[0]
```

Comment:
This seems bad.
I'm not 100% sure what you are doing, but are you trying to target a specific quiz?
If so, then just path to the quiz directly with `tests.base.GOOD_QUIZZES_DIR`.